scipy/fftpack/realtransforms.py

```diff
@@ -611,15 +631,27 @@ def dst(x, type=2, n=None, axis=-1, norm=None, overwrite_x=False):
 
     .. versionadded:: 0.11.0
 
+    **Type IV**
```

Comment:
It would probably be better to format these with something like:
```
Type IV
^^^^^^^
```
to let `Sphinx` do the sub-heading formatting rather than forcing bold. But this is not your fault (you are just following existing convention) so feel free to change it or not